src/algoesup/magics.py

```diff
@@ -228,17 +228,19 @@ def run_checkers(result) -> None:
         with tempfile.NamedTemporaryFile(mode="w", suffix=".py", delete=False) as temp:
             # transform IPython to pure Python to avoid linters reporting syntax errors
             temp.write(TransformerManager().transform_cell(result.info.raw_cell))
-            temp_name = temp.name
+            # Backslash causes esc sequence error from standard Windows file paths,
+            # but Windows accepts both "\" and "/" as separators.
+            temp_name = temp.name.replace("\\", "/")
         for checker in active:
             command, display = checkers[checker]
-            command += " " + temp_name
+            command.append(temp_name)
```

Comment:
Previously, `command` was a string so line 234 created a new string. Now it's a list and the entry in the dict is changed, so each run of a linter adds a new file and ruff complains with 'file no found' because the files get removed. I think the fix is to simply do `lint_file = command + [temp_name]` and use `lint_file` in the subprocess. Please check on Windows and Linux.

Reply:
Thanks for spotting that one. I have made the requested changes, and corrected another instance of the same problem.

All seems to be working on Linux and Windows AFAIK

Reply:
Works on macOS too